src/common/ActionSdkHelper.js

```diff
@@ -0,0 +1,371 @@
+import * as actionSDK from "@microsoft/m365-action-sdk";
```

Comment:
why we have migrated from ActionSdkHelper.ts to ActionSdkHelper.js ?

Reply:
Because we are using HTML and Javascript and want to add some more actionSDK methods for image handling.